streams/src/main/java/org/apache/kafka/streams/processor/internals/AbstractTask.java

```diff
@@ -169,17 +183,18 @@ void registerStateStores() {
                 e
             );
         }
-        log.trace("Initializing state stores");
+        log.debug("Acquired state directory lock");
```

Comment:
We used to have some issues around locking ordering with multi-threading within a JVM in the past, and even now I'm not sure if we've cleared all the bugs, so I'm promoting it for debuggibility just in case.